### PR TITLE
Dont use deprecated methods

### PR DESCRIPTION
**Pull request by Benestar**
This removes usage of lots of deprecated b/c methods in this package and replaces them by the new methods.

<s>This actually depends on #474 </s>

[Bug: T98290](https://phabricator.wikimedia.org/T98290)

### CODE REVIEW

**Comment by thiemowmde** [comment]
-2. As long as https://phabricator.wikimedia.org/T87237 is not solved no new instances of Fingerprint should be introduced.

**Comment by JeroenDeDauw** [comment]
CR+2

File: src/Entity/Diff/PropertyDiffer.php [hunk 1]
```diff
@@ -79,9 +79,9 @@ private function diffPropertyArrays( array $from, array $to ) {
 	private function toDiffArray( Property $property ) {
```
Comment (thiemowmde) [inline src/Entity/Diff/PropertyDiffer.php]: The full property is not needed inside of this method.

Comment (Benestar) [inline src/Entity/Diff/PropertyDiffer.php]: Indeed, a `Fingerprint` is sufficient  here. I already thought about creating a separate `FingerprintDiffer` but I think  these 5 lines of code duplication are not worth a new class.

Comment (JeroenDeDauw) [inline src/Entity/Diff/PropertyDiffer.php]: I concur a new class is not warranted at this point. And I agree a Fingerprint could be passed to this method, though find this an odd observation given the -2.

**Pull request closed without merging**